www/include/common/common-Func.php

```diff
@@ -1112,43 +791,6 @@ function getMyServiceExtendedInfoField($service_id, $field)
     }
 }
 
-function getMyServiceExtendedInfoImage($service_id, $field)
```

Comment:
This funcion is still used in :
- www\include\configuration\configObject\service\listServiceByHost.php:346
- www\include\configuration\configObject\service\listServiceByHostGroup.php:313
- www\include\configuration\configObject\service_template_model\listServiceTemplateModel.php:219
- www\include\monitoring\objectDetails\serviceDetails.php:872
- www\include\monitoring\objectDetails\serviceDetails.php:877